test/cleanup.go

```diff
@@ -88,8 +86,9 @@ func (c *Cleaner) Clean(awaitDeletion bool) error {
 		if err != nil {
 			c.logf("Failed to get to-be cleaned resource %q : %s", deleter.Name, err)
 		} else {
-			bytes, _ := json.MarshalIndent(r, "", "  ")
-			c.logf("Cleaning resource: %q\n%+v", deleter.Name, string(bytes))
+			// bytes, _ := json.MarshalIndent(r, "", "  ")
```

Comment:
Should "Cleaning resource:" still be a prefix on the log?

In any case, remove the commented code.

Reply:
Sorry I mistakenly removed them for local debug, will add them back.